src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -134,3 +134,26 @@ Errors: Error 0-14: Invalid argument type. Expecting one of the following: Numbe
 >> Value(Index(ParseJSON("{""a"": 5, ""b"": [{""c"": 19 }, {""c"": ""foo"" }] }").b, 1).c)
 19
 
+>> DateDiff(DateValue(ParseJSON("""2011-01-15""")), DateValue(ParseJSON("""2011-01-30""")))
+15
+
+>> DateValue(ParseJSON("""2011-1-15"""))
+#Error(Kind=BadLanguageCode)
+
+>> DateValue(ParseJSON("null"))
+Blank()
+
+>> DateValue(ParseJSON("""abcdef"""))
+#Error(Kind=BadLanguageCode)
+
+>> DateDiff(DateTimeValue(ParseJSON("""2011-01-15T08:00:00.000Z""")), DateTimeValue(ParseJSON("""2011-01-30T08:00:00.000Z""")))
+15
+
+>> DateTimeValue(ParseJSON("""2011-01-15T08:0:00.000Z"""))
+#Error(Kind=BadLanguageCode)
```

Comment:
is BadLanguageCode the right error kind here?

Reply:
Changed to InvalidArgument